litellm/utils.py

```diff
@@ -1665,6 +1664,33 @@ def success_handler(
                             end_time=end_time,
                             print_verbose=print_verbose,
                         )
+                    if callback == "logfire":
+                        global logfireLogger
+                        verbose_logger.debug("reaches logfire for success logging!")
+                        kwargs = {}
+                        for k, v in self.model_call_details.items():
+                            if (
+                                k != "original_response"
+                            ):  # copy.deepcopy raises errors as this could be a coroutine
+                                kwargs[k] = v
+
+                        # this only logs streaming once, complete_streaming_response exists i.e when stream ends
+                        if self.stream:
+                            if "complete_streaming_response" not in kwargs:
+                                break
```

Comment:
can we use 'continue' instead of 'break' here - so other logging integrations aren't impacted @elisalimli

Reply:
Good catch! I have realized that we're using break statement not only for the logfire callback, but also others. Therefore, I pushed a fix in [ed7c9e4](https://github.com/BerriAI/litellm/pull/3444/commits/ed7c9e4e8769cc7a6396c50353ee85c366864516)